main.tf

```diff
@@ -17,6 +17,8 @@ resource "azurerm_key_vault" "kv" {
   enabled_for_template_deployment = true
   soft_delete_retention_days      = 90
 
+  enable_rbac_authorization = var.enable_rbac_authorization
```

Comment:
I think the access policies will need to be flagged if enabled?

and role assignments created to replace the access policies?

Reply:
Flagged the Policy if enabled is false and same the other way round.

Reply:
would Key Vault Reader work?
https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#key-vault-reader

will have read access but cannot read sensitive data like secret values?

Reply:
The current permissions are basically 'list' and 'set'.

product teams need to be able to manage the values, retrieving is debate-able, generally they don't need to had a bit of a chat internally about it but it didn't really go anywhere